### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by danperezsan**
PS-125: Add backup_configuration block to private_postgres composite module

### CODE REVIEW

File: gcp/databases/cloudsql/private_postgres/main.tf [hunk 1]
```diff
@@ -29,7 +29,17 @@ module cloudsql-postgres {
     ipv4_enabled = false
     private_network = data.google_compute_network.main.id
     require_ssl = true
-    allocated_ip_range = module.private-service-access.google_compute_global_address_name
+    allocated_ip_range = null
```
Comment (AlexBMes) [inline gcp/databases/cloudsql/private_postgres/main.tf:32]: Why was this changed to null?

Comment (danperezsan) [reply gcp/databases/cloudsql/private_postgres/main.tf:32]: Had to set it to null otherwise terraform was failing with error:
"Cannot update allocated IP range: changing range within a VPC is not allowed"

**Review by AlexBMes — DISMISSED** [review]
